config.json

```diff
@@ -1,6 +1,7 @@
 {
   "language": "Go",
   "active": true,
+  "blurb": "Go is a compiled, open source programming language with a small, consistent syntax, a powerful standard library and fantastic tooling. It's a great fit for web backends and command-line tools.",
```

Comment:
At risk of getting too long and list-y, maybe mention systems & concurrency?

*"Go is a compiled, open source programming language with a small, consistent syntax, a powerful standard library, built-in concurrency primitives and fantastic tooling. It's a great fit for web backends, command-line tools and large-scale systems."*